open-sphere-base/mantle/src/main/java/io/opensphere/mantle/plugin/selection/BufferRegionCreator.java

```diff
@@ -180,7 +183,26 @@ protected void destroyPreview()
     {
         if (myPreviewGeometry != null)
         {
-            unregisterGeometry(myPreviewGeometry);
+            if (myGeometry != null && myGeometry instanceof MultiPolygonGeometry)
```

Comment:
null check not necessary before instanceof

Reply:
fixed